cymetric/cycamore_root_metrics.py

```diff
@@ -0,0 +1,18 @@
+from cymetric import root_metric
+
+#root metrics that come with the archetypes within the cycamore module
+agentstate_cycamore_enrichmentinfo = root_metric(
```

Comment:
Does cymetric have an actual dependency on cycamore? Or is this another testing challenge?